src/ui/button/auth_button.jsx

```diff
@@ -2,10 +2,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 const AuthButton = props => {
-  const { disabled, isBig, label, onClick, strategy, icon, primaryColor, foregroundColor } = props;
+  const { disabled, label, onClick, strategy, icon, primaryColor, foregroundColor } = props;
 
-  let className = 'auth0-lock-social-button';
-  if (isBig) className += ' auth0-lock-social-big-button';
+  let className = 'auth0-lock-social-button auth0-lock-social-big-button';
```

Comment:
className is never reassigned, should use const instead

Reply:
fixed.